test/suites/clustering.sh

```diff
@@ -2756,7 +2756,7 @@ test_clustering_image_refresh() {
   # Modify public testimage
   old_fingerprint="$(LXD_DIR="${LXD_REMOTE_DIR}" lxc image ls testimage -c f --format csv)"
   dd if=/dev/urandom count=32 | LXD_DIR="${LXD_REMOTE_DIR}" lxc file push - c1/foo
-  LXD_DIR="${LXD_REMOTE_DIR}" lxc publish c1 --alias testimage --public
+  LXD_DIR="${LXD_REMOTE_DIR}" lxc publish c1 --alias testimage --reuse --public
```

Comment:
Did you add this because the old behaviour was to reuse by default?

Reply:
Yes.